docs/backup-recovery/README.md

```diff
@@ -120,6 +120,8 @@ The backup workload generates an utility called `upgrade-recovery.sh` in the rec
 
 ## Recovery from Upgrade Failure
 
+In case, the upgrade failed in a spoke cluster, the cgu needs to be deleted in the hub cluster and an admin needs to login to the spoke cluster to start the recovery process.
```

Comment:
I don't see "cgu" anywhere else. I see "TALO CR" on line 92. Make it consistent?